api_app/_version.py

```diff
@@ -1 +1 @@
-__version__ = "0.9.2"
+__version__ = "0.9.3"
```

Comment:
This is an added functionality:
```suggestion
__version__ = "0.10.0"
```